CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#972](https://github.com/nix-rust/nix/pull/972))
 - Added `symlinkat` wrapper.
   ([#997](https://github.com/nix-rust/nix/pull/997))
+- Support for no-change of parameters of `setresuid` and `setresgid` wrapper
+  ([#931](https://github.com/nix-rust/nix/issues/931))
```

Comment:
Link to the pull request, not the issue.

Reply:
Fixed... So used to linking to the issue.